### PR TITLE
Signor complexes

### PR DESCRIPTION
**Pull request by steppi**
This PR adds equivalences to Signor complexes for some existing Famplex complexes. The process of finding equivalences was not completely systematic and it's possible that more equivalences can be found.

### CODE REVIEW

File: export/famplex.obo [hunk 1]
```diff
@@ -15,6 +15,7 @@ inverse_is_a: HGNC:HTR7
 [Term]
 id: FPLX:5_hydroxytryptamine_receptors_ionotropic
 name: 5-hydroxytryptamine-receptors-ionotropic
+def: "A subclass of serotonin receptors that form cation channels and mediate signal transduction by depolarizing the cell membrane. The cation channels are formed from 5 receptor subunits. When stimulated the receptors allow the selective passage of SODIUM; POTASSIUM; and CALCIUM." [mesh:D044406]
```
Comment (cthoyt) [inline export/famplex.obo:18]: oh crazy, somehow I forgot to update the OBO file when I made the PR that added the definitions.

File: equivalences.csv [hunk 2]
```diff
@@ -2068,6 +2068,30 @@ RE,R-HSA-936779,ATP1A
...
+SIGNOR,SIGNOR-C243,IFNAR
+SIGNOR,SIGNOR-C245,Adaptor_protein_II
+SIGNOR,SIGNOR-C247,Adaptor_protein_III
+SIGNOR,SIGNOR-C248,Adaptor_protein_I
+SIGNOR,SIGNOR-C262,DNA_polymerase_alpha
+SIGNOR,SIGNOR-C264,ATP_synthase
+SIGNOR,SIGNOR-C266,MRPS
+SIGNOR,SIGNOR-C304,Cohesin
+SIGNOR,SIGNOR-C313,MAC
+SIGNOR,SIGNOR-C317,Fibrin
```
Comment (cthoyt) [inline equivalences.csv:2094]: Are you doing these by hand?

File: export/famplex.obo (hunk 1 above)
Comment (bgyori) [reply export/famplex.obo:18]: I was just wondering how come the tests didn't error last time, they should normally error on any diffs in exports...

File: equivalences.csv (hunk 2 above)
Comment (cthoyt) [reply equivalences.csv:2094]: If so, maybe we should add SIGNOR's names to PyOBO (https://github.com/pyobo/pyobo/issues/107) then do some curation a la Biomappings style

Comment (steppi) [reply equivalences.csv:2094]: Yeah, I did them basically by hand. I got a dump of the SIGNOR complexes and scanned through it for things that looked familiar. I may have missed a few

**Pull request merged**